config/ogdl/src/main/java/org/apache/shiro/config/ogdl/CommonsInterpolator.java

```diff
@@ -35,12 +35,12 @@
  *     </tr>
  *     <tr>
  *         <td>sys</td>
- *         <td>${sys:os.name}</td>
+ *         <td>${os.name}</td>
  *         <td>mac os x</td>
  *     </tr>
  *     <tr>
  *         <td>env</td>
- *         <td>${env:EDITOR}</td>
+ *         <td>${EDITOR}</td>
```

Comment:
What do you think about
```suggestion
 *         <td>${EDITOR} (resolved from environment variables)</td>
```

Reply:
perfect.